python/datasources/cdc_restricted.py

```diff
@@ -113,13 +127,35 @@ def generate_breakdown(self, df, demo, geo):
             std_col.COVID_POPULATION_PCT,
         ]
 
+        if geo == 'national' or cumulative:
+            # We don't want the missing values for the time series data
+            # but we do need them to correctly calculate the national
+            # ALL per_100k rates
```

Comment:
I think i need you to explain this in more detail; i still don't totally understand what's happening here

Reply:
When a racial group doesn't have any COVID cases for a state or county we still want them to show up on the on the table card. Also, when we calculate the national COVID numbers we want racial groups that don't have any COVID cases in a state count in that state's population when we calculate the national per_100k rates. However I don't want to inflate the size of the time_series state and county files, where we don't need the missing data as you do the calculation on the frontend to fill in nulls.